SConstruct

```diff
@@ -172,6 +175,19 @@ def render_thumbnail(env, target, source):
                                                  resolution=env['RESOLUTION'],
                                                  appleseed_path=APPLESEED_PATH)
 
+    else:
+        process = batchtools.sbsrender_render(
+            os.path.abspath("./data/dependencies/pbr_render.sbsar"),
+            output_name=os.path.splitext(os.path.basename(str(target[0])))[0],
+            output_path=os.path.dirname(str(target[0])),
+            engine="ogl3" if platform.system() == "Linux" else "d3d10pc",
```

Comment:
Flip this to look for Windows rather than linux and we'll get Mac support out of the box.
Please move it to a early shared variable called something like GPU_ENGINE too and add it to the render_map too so we get GPU support there too!